sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingSessionWindowSuite.scala

```diff
@@ -50,8 +51,7 @@ class StreamingSessionWindowSuite extends StreamTest
       (SQLConf.STATE_STORE_PROVIDER_CLASS.key, value.stripSuffix("$"))
     }
     // RocksDB doesn't support Apple Silicon yet
-    if (System.getProperty("os.name").equals("Mac OS X") &&
-        System.getProperty("os.arch").equals("aarch64")) {
+    if (Utils.isMac && System.getProperty("os.arch").equals("aarch64")) {
```

Comment:
I piggy-back this replacement from `System.getProperty("os.name").equals("Mac OS X")` to `Utils.isMac`.